script/ops/Ops_2024_08_22.s.sol

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.18;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { RewardForwarder } from "cove-contracts-boosties/src/rewards/RewardForwarder.sol";
+import { ERC20RewardsGauge } from "cove-contracts-boosties/src/rewards/ERC20RewardsGauge.sol";
+import { CoveYearnGaugeFactory } from "cove-contracts-boosties/src/registries/CoveYearnGaugeFactory.sol";
+import { CoveToken } from "cove-contracts-boosties/src/governance/CoveToken.sol";
+import { console2 as console } from "forge-std/Console2.sol";
+import { OpsMultisigScript } from "./OpsMultisigScript.s.sol";
+
+contract Script is OpsMultisigScript {
+    function run(bool shouldSend) public override {
+        super.run(shouldSend);
+
+        address coveToken = deployer.getAddress("CoveToken");
+        address coveYfiRewardsGauge = deployer.getAddress("CoveYfiRewardsGauge");
+        address coveYfiRewardsGaugeRewardForwarder = deployer.getAddress("CoveYFIRewardsGaugeRewardForwarder");
+        CoveYearnGaugeFactory factory = CoveYearnGaugeFactory(deployer.getAddress("CoveYearnGaugeFactory"));
+        CoveYearnGaugeFactory.GaugeInfo[] memory info = factory.getAllGaugeInfo(9, 0);
```

Comment:
Same mistake as the last script. Lets change this to 100 and add some sanity checks that can catch this at the end. (ensuring the total amount of COVE tokens == differences from beginning and the end)

Reply:
ahh same thing as my last multisig pr 😢